web/src/helpers/topos.js

```diff
@@ -5,13 +5,10 @@ const constructTopos = () => {
   const zones = {};
   Object.keys(topo.objects).forEach((k) => {
     if (!topo.objects[k].arcs) { return; }
-    let geo;
-    // Do merge inner arcs for those
-    if (['US'].indexOf(k.split('-')[0]) !== -1) {
-      geo = topojson.feature(topo, topo.objects[k]);
-    } else {
-      geo = { geometry: topojson.merge(topo, [topo.objects[k]]) };
-    }
+    const geo = {
+      geometry: topojson.merge(topo, [topo.objects[k]]),
+      properties: topo.objects[k].properties
+    };
     // Exclude zones with null geometries.
     if (geo.geometry) {
```

Comment:
This check would also not be necessary any longer. If this function excludes any geometries, we might also not want it to exclude silently as this suggests the world.json is corrupted

Reply:
Nice! 👏